src/org/labkey/test/tests/AdminConsoleTest.java

```diff
@@ -151,6 +154,73 @@ public void testRibbonBar()
         assertElementNotPresent(ribbonLink);
     }
 
+    // Issue 51843  allow site banner configuration via api
+    @Test
+    public void testSiteBannerAPIConfiguration() throws Exception
+    {
+        goToAdminConsole();
+
+        String bannerMessage = "test banner message" + TRICKY_CHARACTERS;
```

Comment:
Consider using the injection test string here as well.
```suggestion
        String bannerMessage = "test banner message" + TRICKY_CHARACTERS + INJECT_CHARS_1;
```